README.md

```diff
@@ -68,6 +68,7 @@ Current missing episodes:
 - Season 16: 171-195, 207
 - Season 18: 262-263
 - Season 24: 453-456
+- Season 34 - 968-1085
```

Comment:
should follow same bullet point structure here for consistency: `Season 34: 968-1085`

Reply:
Whoops, nice catch! Fixed